tests/frontend/travis/runner.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+if [ -z ${SAUCE_USERNAME} ]; then echo "SAUCE_USERNAME is unset - exiting"; exit 1; fi
```

Comment:
Use double quotes around expanded bash variables: `"{SAUCE_USERNAME}"`

Reply:
Ack

Reply:
looking at other references it' ``"${SAUCE_USERNAME}"``

You sure on ``"{SAUCE_USERNAME}"``?